hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -469,6 +469,27 @@ public boolean forceExitSafeMode() throws IOException {
     return scm.exitSafeMode();
   }
 
+  @Override
+  public void startReplicationManager() {
+    AUDIT.logWriteSuccess(buildAuditMessageForSuccess(
+        SCMAction.START_REPLICATION_MANAGER, null));
+    scm.getReplicationManager().start();
+  }
+
+  @Override
+  public void stopReplicationManager() {
+    AUDIT.logWriteSuccess(buildAuditMessageForSuccess(
+        SCMAction.STOP_REPLICATION_MANAGER, null));
+    scm.getReplicationManager().stop();
```

Comment:
Shall we want to return a boolean flag which shows whether the stop is a success or not?
Like case calling stop on already stopped replication monitor, we can print like Replication Monitor is not running, to stop it(Or some message like this). 

Same applicable for start also.

Reply:
Stop is an idempotent call and also it's async, so getting status of ReplicationManager immediately after calling stop might some time return true (running).

> Like case calling stop on already stopped replication monitor, we can print like Replication Monitor is not running

If we do this, we can end up in a minor inconsistency for a short period of time. Where stop will say that ReplicationManager is not running, but the status command will say that ReplicationManager is running.

Reply:
This is true, even if we don't return status, as stop() is not a blocking call.
Can we print some output message for the command for stop like we signalled for stop, and if immediately status is called you can still see it as running. (Or something more clearer, to let users know the behavior)

Reply:
Sure.

Reply:
I just see it is mentioned Stopping ReplicationManager... With this info, the user will not understand this is async behavior and immediately calling getStatus he might still see it is running.

Reply:
Updated the message.